### PR TITLE
Feature/auto-dockerhub-images

### PR DESCRIPTION
**Pull request by damccull**
This change updates the docker files to build the current source and update phoenix wallet each launch, so that it is always up to date even when there is no new BRS update.

Use this by creating a new "environment" under the repo settings called `dockerhub-publish` and, inside that environment, setting 2 secrets: `DOCKERHUB_USERNAME` and `DOCKERHUB_PASSWORD` for the username and password of the dockerhub signum account.

Whenever a new Release is created, this should run automatically and create a docker image of the release's corresponding tag, and automatically upload it to dockerhub.

It will docker tag the images as `vX.Y.Z-h2`, `vX.Y.Z-maria`, `latest-h2` and `latest-maria`.

If merged, solves #493

### CODE REVIEW

**Comment by damccull** [comment]
This is just about ready. On a technical standpoint it works really well. Just trying to track down a couple bugs with the phoenix interface. Seems to be missing the signum and bitcoin money symbols, but isn't reporting 404 for any assets.

File: .github/workflows/dockerhub.yml [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    name: Push h2 db Docker image to Docker Hub
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out the repo
+        uses: actions/checkout@v2
+      
+      - name: Log in to Docker Hub
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
```
Comment (ohager) [inline .github/workflows/dockerhub.yml:19]: where do this come from? which credentials are being used? Do we need to add them to github secrets?

File: .github/workflows/dockerhub.yml [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+            latest=false
+      
+      - name: Build and push Docker image
+        uses: docker/build-push-action@v2
+        with:
+          file: 'Dockerfile.mariadb'
+          context: .
+          push: true
+          tags: ${{ steps.meta.outputs.tags }}
+          labels: ${{ steps.meta.outputs.labels }}
```
Comment (ohager) [inline .github/workflows/dockerhub.yml:73]: no newline at end...

**Review by ohager — COMMENTED** [review]
Great Job. Haven't tested it yet....will do in a few moments.

File: .github/workflows/dockerhub.yml (hunk 1 above)
Comment (damccull) [reply .github/workflows/dockerhub.yml:26]: Added comments to the top explaining this.

File: .github/workflows/dockerhub.yml (hunk 2 above)
Comment (damccull) [reply .github/workflows/dockerhub.yml]: Newline added.

File: .github/workflows/dockerhub.yml (hunk 1 above)
Comment (damccull) [reply .github/workflows/dockerhub.yml:26]: Added instructions to the top of the yml file

**Comment by damccull** [comment]
If merged, closes #493

**Pull request merged**